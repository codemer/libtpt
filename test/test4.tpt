This test also includes a file.
@set(inc,"test4inc.tph")\
@include(inc)\
But this include should have used a variable.
