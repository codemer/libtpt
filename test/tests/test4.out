This test also includes a file.
var2 = The red fox runs through the plain and jumps over the fence.
But this include should have used a variable.
